superset/migrations/shared/migrate_viz/processors.py

```diff
@@ -53,3 +86,30 @@ def _pre_action(self) -> None:
         if x_axis_label:
             self.data["x_axis_title"] = x_axis_label
             self.data["x_axis_title_margin"] = 30
+
+
+class MigrateLineChart(TimeseriesChart):
+    source_viz_type = "line"
+    target_viz_type = "echarts_timeseries_line"
+    rename_keys = {
+        "x_axis_label": "x_axis_title",
+        "bottom_margin": "x_axis_title_margin",
+        "x_axis_format": "x_axis_time_format",
+        "y_axis_label": "y_axis_title",
+        "left_margin": "y_axis_title_margin",
+        "y_axis_showminmax": "truncateYAxis",
+        "y_log_scale": "logAxis",
+    }
+
+    def _pre_action(self) -> None:
+        super()._pre_action()
+
+        line_interpolation = self.data.get("line_interpolation")
```

Comment:
Also surprised that the auto-walrus pre-commit hook didn't change this.

Reply:
Same as above.